Update documentation with textual improvements (#1170)
asreview/asreview: docs/source/progress.rst

- Labeled records: the number of records that you labeled as relevant or irrelevant, including those you added as prior knowledge.
- Relevant records: the number of records that you labeled as relevant, including those you added as prior knowledge.
- Irrelevant records: the number of records that you labeled as irrelevant, including those you added as prior knowledge.
- Irrelevant since last relevant: the number of irrelevant records you have seen since the last relevant record. 

Charts
~~~~~~

The charts on the analytics page can be useful to monitor your progress. There
is a *Progress* and a *Recall* chart. The charts do not include prior

> knowledge and are most relevant after you have screened a couple of records.

Reviewer: ```suggestion
knowledge and are most relevant after you have screened a couple of records.
```

How many records? I think this should be at least 10 to make the progress chart work.

**Recall chart**

The recall chart plots the number of relevant records against the number of
records that you reviewed in ASReview LAB. *Relevant by ASReview LAB* refers to
the relevant records that you labeled with the assistance of the active
learning model. *Random relevant* refers to the relevant records that you might
find if you manually reviewed the records so far without the assistance of the
active learning model.

> **Export Figure**

Reviewer: Nice!!

<https://github.com/asreview/asreview/discussions/557>`_ discussion provide
tips on when to stop with screening.

.. tip::

>   The number of irrelevant records since last relevant will increase the longer you screen.

Reviewer: ```suggestion
  The number of *irrelevant records since last relevant* will increase the longer you screen.
```

.. tip:: 

>   If you selected *Maximum* as the :ref:`project_create:Query Strategy`, you will 

Reviewer: ```suggestion
  With *Maximum* as :ref:`project_create:Query Strategy`, you will 
```

Export results
--------------

> You can export the results of your labeling decissions to a RIS, CSV, TSV or Excel file.

Reviewer: ```suggestion
You can export the results of your labeling to a RIS, CSV, TSV, or Excel file.
```
